scripts/ci-conformance.sh

```diff
@@ -84,7 +84,7 @@ if grep -iqF "$(echo "${AWS_ACCESS_KEY_ID-}" | \
 fi
 
 # TODO(dims): remove "SKIP_INIT_IMAGE" once we fix problem building images in CI
-SKIP_INIT_IMAGE=yes hack/ci/e2e-conformance.sh --verbose
+hack/ci/e2e-conformance.sh --verbose --skip-init-image --use-ci-artifacts
```

Comment:
Wouldn't we want to limit the use of `--use-ci-artifacts` to just the "stable" job?

Reply:
@detiber y, there's more tweaking to do here. i want to be able to specify this in the CI job definition itself as well. For now, i want to enable this across all the jobs to see how it holds up over the weekend